hpedockerplugin/volume_manager.py

```diff
@@ -683,9 +686,12 @@ def _create_snapshot(self, src_vol_name, schedName, snapshot_name,
         vol['snapshots'].append(db_snapshot)
         snap_vol['snap_metadata'] = db_snapshot
         snap_vol['backend'] = current_backend
+        snap_vol['3par_vol_name'] = bkend_snap_name
```

Comment:
Shall we remove this injection during create operation?
We can have 3par_vol_name injected in inspect operation, doing this will take care of all the use cases like create volume, clone, snapshot and import.

Reply:
while creation the output of function definition is not captured previously, now it's captured in a variable `bkend_vol_name` and used for populating the fields for snapshot details/volume details. I don't want that utils.get_3par_name() call to invoked again, when there is already a computed value for this (during snapshot/volume/clone creation process).